cinema/management/commands/wait_for_db.py

```diff
@@ -0,0 +1,19 @@
+import time
+
+from django.core.management.base import BaseCommand, CommandError
+from django.db import connections
+from django.db.utils import OperationalError
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        self.stdout.write("Waiting for database")
+        connect = None
+        while not connect:
+            try:
+                connect = connections["default"]
+                connect.cursor()
```

Comment:
You are getting the connection and immediately after you are creating a cursor. This is unnecessary if you only want to check the connection. Creating a cursor might fail for reasons other than the connection being not ready. It would be better to only get the connection in the try block.